lib/logstash/devutils/rspec/spec_helper.rb

```diff
@@ -17,6 +17,10 @@
 require "logstash/devutils/rspec/logstash_helpers"
 require "insist"
 
+unless LogStash::Environment.const_defined?(:LOGSTASH_HOME)
+  LogStash::Environment::LOGSTASH_HOME = File.expand_path("../../../../../", __FILE__)
```

Comment:
I like the idea of having this in here, however I don't see how this
```File.expand_path("../../../../../", __FILE__)``` is pointing the LOGSTASH_HOME to a relevant place.

Are we sure all the time this way of pointing the variable is going to be consistent ? 

Where do we want to point LOGSTASH_HOME ? as for the recent changes I undersand we aim to do it to the plugin home directory, isn't?